test/integration/command-monitoring/command_monitoring.spec.test.ts

```diff
@@ -0,0 +1,16 @@
+import * as path from 'path';
+
+import { loadSpecTests } from '../../spec';
+import { runUnifiedSuite } from '../../tools/unified-spec-runner/runner';
+
+// This is skipped because our command monitoring happens at the connection
+// level and is using the server reply for the single insert that the bulk
+// performed since it was only one document in the test. The test expectation
+// is that we are using the bulk write result which was returned to the user
+// as the reply in the command succeeded event instead of our raw reply from
+// the server. There's nothing we can change here.
+const SKIP = ['A successful unordered bulk write with an unacknowledged write concern'];
```

Comment:
do we have coverage for this scenario in some other way elsewhere? or can we substitute our own replacement version of the test by just copy/pasting the json and modifying the expectation?

Reply:
Well this scenario isn't actually valid for us. The command monitoring spec does not specify where in the driver's object hierarchy it needs to be implemented, nor does it specify if the reply in the command succeeded event is the raw response from the server or the result that the user would see. Drivers were free to implement this as they choose and this test. while valid for Swift and others, isn't valid for Node or Ruby for example. Since this spec test is for command monitoring and not crud, it didn't seem to be covered by another test here - the crud unified tests have assertions for the unacknowledged scenarios.

Reply:
ok that seems fine to me